beater/config/config.go

```diff
@@ -42,6 +44,40 @@ var (
 	regexObserverVersion = regexp.MustCompile("%.*{.*observer.version.?}")
 )
 
+type kibanaConfig struct {
+	Enabled             bool `config:"enabled"`
+	kibana.ClientConfig `config:",inline"`
+}
+
+func (k *kibanaConfig) Unpack(cfg *common.Config) error {
+	if cfg == nil {
+		return nil
+	}
```

Comment:
Pretty sure this is unnecessary. go-ucfg's `Config.Unpack` will return an error if the config is nil, before calling into the provided value.

Reply:
I noticed you removed it in the last suggestion but hesitated since this is exported.  On second thought, taking the suggestion.